common/on_actions/00_on_actions.txt

```diff
@@ -1320,6 +1320,9 @@ on_decade_pulse = {
 			40 = HF.6 # A Prophecy foretold
 			890 = 0
 		}
+		
+		# Warcraft
+		30 = WCOME.600 #Chance to convert to Old God religion if you control the prison
```

Comment:
You could just increase the chance of `WCVOD.9900` (random Void event).

Reply:
yeah but you told me to add @Kaisustrin 's suggestion

Reply:
> yeah but you told me to add @Kaisustrin 's suggestion

I thought you knew we have random events to convert to the Old God's religion already.

Reply:
i know about them but since the suggestion was to add a new one i did just that